### PR TITLE
feat: added basic routing

### PR DESCRIPTION
**Pull request by SinchanaMS**
Added basic routing to the Video Library app.

### CODE REVIEW

**Review by ShrutiLakhani — APPROVED** [review]
Everything looks good. You have structured the folders well. You can go ahead and merge.

File: src/pages/pages.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import History from "./History/History";
+import Login from "./Authentication/Login"
+import SignUp from "./Authentication/SignUp"
+import Liked from "./Liked/Liked";
+import MockApi from "./Mockman/MockApi";
+import PageNotFound from "./PageNotFound/PageNotFound";
+import Playlist from "./Playlist/Playlist";
+import WatchLater from "./WatchLater/WatchLater";
+
+export {Explore, History, Login, SignUp, Liked, MockApi,PageNotFound, Playlist, WatchLater}
```
Comment (muskanbagrecha) [inline src/pages/pages.js:11]: Named exports allow us to export directly. So we don't have to import it first and then export it like how you've done. 
```javascript
export { Login } from "./Authentication/Login"
```
You can play with it later.

**Review by muskanbagrecha — APPROVED** [review]
The folder structure looks good. You can consider using ES6+ syntax for your components. Looks good to merge otherwise.

File: src/pages/pages.js (hunk 1 above)
Comment (SinchanaMS) [reply src/pages/pages.js:11]: Oh! I wasn't aware of this. Thank you!

**Pull request merged**